content/en/network_performance_monitoring/installation.md

```diff
@@ -122,14 +126,52 @@ If you need to use Network Performance Monitoring on other systems with SELinux
 
 If these utilities do not exist in your distribution, follow the same procedure but using the utilities provided by your distribution instead.
 
+### Windows systems
+
+Data collection for Windows systems is done using a Microsoft-certified Windows driver, which is now available in public beta for versions 2016 or later. Note that the driver currently monitors Windows hosts only, and not Windows containers. DNS metric collection is not currently supported for Windows systems.
+
+To enable network performance monitoring with the Windows driver:
+
+1. Install [this custom build][9] of the Datadog Agent 
+2. Edit `/etc/datadog-agent/system-probe.yaml` to set the enable flag to `true`:
+
+    ```yaml
+    system_probe_config
+    :
+        ## @param enabled - boolean - optional - default: false
+        ## Set to true to enable the System Probe.
+        #
+        enabled: true
+    ```
+    ```
+3. Edit `/etc/datadog-agent/datadog.yaml` to set the enable flag to `true`:
+
+    ```yaml
+    process_config
+    :
+        ## @param enabled - boolean - optional - default: false
+        ## Set to true to enable the System Probe.
+        #
+        enabled: true
+    ```
+    ```
+4. [Restart the Agent][2]
+
+    ```shell
+    sudo systemctl restart datadog-agent
```

Comment:
yeah, that's not going to work so well on windows :)

Command shell: net /y stop datadogagent && net start datadoagagent
Powershell: restart-service -f datadogagent